packages/buttongroup/README.md

```diff
@@ -0,0 +1,89 @@
+# @zendeskgarden/container-buttongroup [![npm version](https://img.shields.io/npm/v/@zendeskgarden/container-buttongroup.svg?style=flat-square)](https://www.npmjs.com/package/@zendeskgarden/container-buttongroup)
+
+This package includes containers relating to buttongroup in the
```

Comment:
This generated text reads a bit weird. Maybe "...relating to button groups..." instead?